### DISCUSSION TITLE
Previous successful submission changed to error occurred

### DISCUSSION THREAD

**Discussion by shaido987** [PaddlePaddle/PaddleSpatial#138] [BaiduKDDCup2022] [answered]
Hello,

I have two previously successful submissions that got quite good scores. Today they suddenly changed to "Error Occurred". I saw that the top-scored submissions are re-run periodically so it could be due to this together with some change in the code? 

The error shown is just "异常", could I get some more info what went wrong?

Team: didadida_hualahuala
Submission (both should have the same error I guess):

![image](https://user-images.githubusercontent.com/1130029/171602034-ee043677-0749-4cb4-9f23-675f8e21c1fa.png)

**Comment by ramber1836** [comment, marked as answer]
When the previous successful submissions are selected for re-evaluation, the status will be ambiguous.
If during the re-evaluation, the competitors check their submissions, the system might return some error messages resulting from the timeout issue, which is a temporary status. After the re-evaluation is finished, the detailed status information (success with a score or failure with some error msgs) will return.
